src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java

```diff
@@ -89,6 +89,12 @@ private void validateHistoryTypeField(Integer type) {
 	}
 
 	private void validateUserId(String user_id) {
+
+		if (user_id == null){

```

Comment:
This won't work due to equals comparison with `==` does not work with `String` objects. 
I remember we had a Validation util method that executes a null exception. Please review it.

Reply:
yes! I think i resolved.